fe/fe-core/src/main/java/com/starrocks/planner/PlanNode.java

```diff
@@ -956,4 +956,9 @@ public List<SlotId> getOutputSlotIds(DescriptorTable descriptorTable) {
     // 3. HashJoinNode: slotIds of both sides of Join equal conditions in semi join and inner join.
     public void collectEquivRelation(FragmentNormalizer normalizer) {
     }
+
+    // disable optimize depends on physical order
+    // eg: sortedStreamingAGG/ PerBucketCompute
+    public void disablePhysicalDistributionOptimize() {
```

Comment:
This name isn't reasonable, PhysicalDistribution general means shuffle, broadcast, colocate.

Reply:
@kangkaisen  Any better name?

Reply:
PhysicalProperty?